ipaas-profile/dbtosf_project/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "kavi"
+name = "dbtosf_project"
```

Comment:
@a5anka do we need to upgrade this to latest ballerina version?

Reply:
These template files are generated by Choreo when we select the Ballerina project preset